mlflow/utils/autologging_utils/__init__.py

```diff
@@ -521,12 +521,24 @@ def autologging_is_disabled(integration_name):
     return False
 
 
+def is_autolog_supported(integration_name):
```

Comment:
```suggestion
def is_autolog_supported(integration_name: str) -> bool:
```

let's add a type hint.